grass7/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.py

```diff
@@ -108,6 +108,17 @@
 #% guisection: Filter
 #%end
 #%option
+#% key: sleep
+#% description: Sleep time in second before retrying to download data from ESA LTA
```

Comment:
I wonder if a definition given in minutes would be more appropriate here...

Reply:
It could make sense, going to modify code and docs

Reply:
done